test/trace.test.js

```diff
@@ -0,0 +1,5 @@
+'use strict'
+
+const t = require('tap')
+require('./helper').payloadMethod('trace', t)
```

Comment:
this should help to unsterstand what is going on

```suggestion
require('./helper').payloadMethod('trace', t, true)
```